unaligned_bam_to_bqsr_and_vcf/mark_duplicates.cwl

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "mark duplicates"
+baseCommand: ["/usr/bin/java", "-Xmx16g", "-jar", "/usr/picard/picard.jar", "MarkDuplicates"]
+arguments:
+    ["O=", { valueFrom: $(runtime.outdir)/DuplicatesMarked.bam },
+    "ASSUME_SORT_ORDER=", "queryname",
+    "METRICS_FILE=", "mark_dups_metrics.txt",
+    "QUIET=", "true",
+    "COMPRESSION_LEVEL=", "0",
```

Comment:
If we continue to write output to disk, we should remove this.  The alternative is to implement a stream through /dev/stdout and /dev/stdin.  We can discuss in the morning.